pkg/forwarder/sync_forwarder.go

```diff
@@ -42,6 +46,15 @@ func (f *SyncForwarder) sendHTTPTransactions(transactions []*transaction.HTTPTra
 	for _, t := range transactions {
 		if err := t.Process(context.Background(), f.client); err != nil {
 			log.Errorf("SyncForwarder.sendHTTPTransactions: %s", err)
+
+			// If there is an error, instantiate a new HTTP client and retry one time
+			// The new HTTP client is instantiated in case the connection has been closed
```

Comment:
@remeh might be able to suggest a more precise comment that better describes why  we are doing this